study.md

```diff
@@ -50,7 +50,22 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+kitchen {
+  "Cupboard": {
+    "bowls": 3,
+    "plates": 5
+  },
+  "Electric Stove": {
+    "heat1": "small",
+    "heat2": "medium",
+    "heat3": "large",
+    "heat4": "extra large"
+  },
+  "Herbs and Spice Rack": {
+    "Rack1": "chives, lemongrass, mint",
```

Comment:
You would want these items to be in an array. As you have it now, they are all in one string.